staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md

```diff
@@ -1,15 +1,14 @@
 # Azure Active Directory plugin for client authentication
 
-This plugin provides an integration with Azure Active Directory device flow. If no tokens are present in the kubectl configuration, it will prompt a device code which can be used to login in a browser. After login it will automatically fetch the tokens and stored them in the kubectl configuration. In addition it will refresh and update the tokens in configuration when expired.
-
+This plugin provides an integration with Azure Active Directory device flow. If no tokens are present in the kubectl configuration, it will prompt a device code which can be used to login in a browser. After login it will automatically fetch the tokens and store them in the kubectl configuration. In addition it will refresh and update the tokens in the configuration when expired.
 
 ## Usage
 
-1. Create an Azure Active Directory *Web App / API* application for `apiserver` following these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-app-registration)
+1. Create an Azure Active Directory *Web App / API* application for `apiserver` following these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-app-registration). The callback URL of your App needs to be set to the external URL of your Kubernetes API Server.
```

Comment:
This surprises me… we don’t actually support redirecting to the apiserver for anything, right? @colemickens, was this how you configured stuff when you were adding this?

Reply:
I think @stuartleeks told me that it's needed. Might be to identify the caller not for actually calling back?

Reply:
@puja108 - apologies for any confusion. I don't believe that the callback URL is used in the auth flow that is used for RBAC, *but* Azure AD requires that something is specified in the field when you create the app so using the api server seems like a reasonable value as a way to mentally create a connection :-)

Reply:
Getting back to this, as this seem to be the last open comment. I tested this and it does not matter, just is nice to have a connection. However, in our case for example they want to use the same apps for a bunch of clusters, so I'll just change this so it says that the Callback URL does not matter, and you can put whatever you want that helps you make a connection.